### PR TITLE
[MIG] crm_location_nuts: Migration to 10.0

### PR DESCRIPTION
**Pull request by chienandalu**
Depends on:

- [x]  https://github.com/OCA/partner-contact/pull/432

cc @Tecnativa

### CODE REVIEW

File: crm_location_nuts/README.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
+===================
```
Comment (pedrobaeza) [inline crm_location_nuts/README.rst:5]: Use new README template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: crm_location_nuts/README.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/10.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/partner-contact/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/partner-contact/issues/new?body=module:%20base_location_nuts%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (pedrobaeza) [inline crm_location_nuts/README.rst]: Remove this part according template

File: crm_location_nuts/README.rst [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (pedrobaeza) [inline crm_location_nuts/README.rst]: https

File: crm_location_nuts/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    'version': '10.0.1.0.0',
+    'depends': [
+        'crm',
+        'sales_team',
+        'base_location_nuts',
+    ],
+    'data': [
+        'views/crm_lead_view.xml',
+    ],
+    'author': 'Antiun Ingeniería S.L., '
```
Comment (pedrobaeza) [inline crm_location_nuts/__manifest__.py]: Remove Antiun

File: crm_location_nuts/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+              'Tecnativa'
+              'Odoo Community Association (OCA)',
+    'website': 'http://www.antiun.com',
```
Comment (pedrobaeza) [inline crm_location_nuts/__manifest__.py]: Put `https://www.tecnativa.com`

File: crm_location_nuts/models/crm_lead.py [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+# Copyright 2015 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class CrmLead(models.Model):
+    _inherit = 'crm.lead'
+
+    region = fields.Many2one(comodel_name='res.partner.nuts', string="Region")
```
Comment (pedrobaeza) [inline crm_location_nuts/models/crm_lead.py]: @rafaelbn why this is called this way and there are only 2 levels instead of four as in base_location_nuts?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: crm_location_nuts/models/crm_lead.py (hunk 6 above)
Comment (chienandalu) [reply crm_location_nuts/models/crm_lead.py]: @rafaelbn Can you attend this comment?

Comment (rafaelbn) [reply crm_location_nuts/models/crm_lead.py]: @chienandalu because that 2 levels are the only two relevant for address purpose

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@rafaelbn please review

**Comment by chienandalu** [comment]
@rafaelbn Your review is still needed here ;)

File: crm_location_nuts/README.rst [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+===================
+NUTS Regions in CRM
+===================
+
+This module adds the concept of NUTS region to the CRM main objects (lead,
+opportunity), and propagate them through the workflow.
+
+Usage
+=====
+
```
Comment (rafaelbn) [inline crm_location_nuts/README.rst:14]: Please add usage section

**Review by rafaelbn — CHANGES_REQUESTED** [review]
Doesn't work. Tested in runbot:

- Go to lead and create a new one
- Add NUTS values
- Convert lead to opportunity and create a res.partner in the workflow
- BUG: Partner is created **without** the NUTS values

File: crm_location_nuts/models/crm_lead.py (hunk 6 above)
Comment (rafaelbn) [reply crm_location_nuts/models/crm_lead.py]: @chienandalu @pedrobaeza testing this in runbot it doesn't work. So I think it must be imlpemented as in `base_location_nuts`. Could you fix it ? Thanks and sorry for the answer I must test it before.

**Comment by rafaelbn** [comment]
My comment : https://github.com/OCA/crm/pull/161#pullrequestreview-71674885 is FIXED! Thanks!

But @chienandalu this modules doesn't work. In `base_location_nut`if I fill NUTS L4 then L3, L2 and L1 are automatically filled. But in `crm_location_nut` not. This modules doesn't work ...

**Comment by chienandalu** [comment]
@rafaelbn Ok, it didn't have that functionality. Now it does. Anyway, maybe in the future base_location_nuts should be an abstract.

**Comment by pedrobaeza** [comment]
@rafaelbn please re-review this

**Comment by chienandalu** [comment]
@rafaelbn Your review is still needed :)

**Comment by pedrobaeza** [comment]
@rafaelbn please review for closing this

**Comment by rafaelbn** [comment]
@pedrobaeza @chienandalu I cannot test it, runbot isn't available. The is a big queue of "Pending: 27" builds.

**Comment by pedrobaeza** [comment]
You can test it on the customer

**Review by rafaelbn — APPROVED** [review]
Tested in customer production enviroment 👍 

Thanks @chienandalu

**Pull request merged**